src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -1,9 +1,15 @@
 <template>
   <FilingTemplate class="consent-continuation-out" :filing="filing" :index="index">
     <template #body>
-      <div v-if="isFilingComplete">
-        <p class="mt-4">
-          This consent is valid <strong>until {{ expiry }} at 12:01 am Pacific time</strong>.
+      <div>
+        <p class="mt-4" v-if="isFilingComplete && !isConsentExpired">
+          This consent to continue out to {{foreignJurisdiction}} is valid <strong>until {{ expiry }} at 23:59 pm
+          Pacific time</strong>.
+        </p>
```

Comment:
Ledger item:
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/67e974b6-300e-4296-b119-b3b2d88301f4)

Reply:
Please update this screenshot with a sample date and time actually from the API (ie, should be 11:59 PM, I think).

Reply:
Here is the updated screenshot:
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/0ad58308-4759-4a8e-9fee-d912e9cc7afc)

Reply:
Very nice :)